Northstar.CustomServers/mod/scripts/vscripts/_items.nut

```diff
@@ -10094,7 +10094,7 @@ void function InitUnlockAsEntitlement( string itemRef, string parentRef, int ent
 		unlock = file.entitlementUnlocks[fullRef]
 	}
 
-	unlock.entitlementIds.append( entitlementId )
+	unlock.entitlementIds.append( 1 ) // Using `1` here instead of the huge DLC check I done previously seems to keep everything unlocked for real this time
```

Comment:
Technically this now turns the passed `entitlementId` arg into an unused value which isn't super ideal but at the same time, based on commit history, I assume this is a function from vanilla so probably fine in that case? ^^

Reply:
So far game haven't complained about it, and it keeps these extra stuff from DLCs unlocked even when progression is enabled, it also doesn't affect other things that normally would unlock through challenges or gifted via Advocates.

Reply:
The only real issue I've encountered is the cosmetics temporarily are cleared when entering into a server.

Reply:
That's caused by the server not having the patch yet.